pkg/guacrest/server/retrieveDependencies.go

```diff
@@ -1,34 +1,49 @@
+//
+// Copyright 2024 The GUAC Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 package server
 
 import (
 	"context"
 	"fmt"
+	"net/url"
 
-	"github.com/Khan/genqlient/graphql"
 	gql "github.com/guacsec/guac/pkg/assembler/clients/generated"
 	assembler_helpers "github.com/guacsec/guac/pkg/assembler/helpers"
-	gen "github.com/guacsec/guac/pkg/guacrest/generated"
 	"github.com/guacsec/guac/pkg/guacrest/helpers"
-	"github.com/guacsec/guac/pkg/guacrest/pagination"
 	"github.com/guacsec/guac/pkg/logging"
+
+	"github.com/Khan/genqlient/graphql"
 	"golang.org/x/exp/maps"
 )
 
-// node is implemented by all graphQL client types
+// node is implemented by all GraphQL client types
 type node interface {
 	GetId() string
 }
 
 // edgeGen defines the edges used by the transitive dependencies graph traversal.
 type edgeGen interface {
-	// getDirectDependencies returns the nouns that are direct dependencies of the input noun.
+	// getDirectDependencies returns the nodes that are direct dependencies of the input noun.
 	getDirectDependencies(ctx context.Context, v node) ([]node, error)
 
-	// getEquivalentNodes returns the nouns that are considered equivalent to the input noun.
+	// getEquivalentNodes returns the nodes that are considered equivalent to the input noun.
 	getEquivalentNodes(ctx context.Context, v node) ([]node, error)
 }
 
-// byDigest is an edgeGen that observes relationships between nouns when they are
+// byDigest is a edgeGen that observes relationships between noun when they are
```

Comment:
nit: I don't think these updates are correct

Reply:
Yeah, that's my bad. I removed the comment by mistake and then re-typed it manually... I have fixed it.